jobs/rocoto/postsnd.sh

```diff
@@ -1,9 +1,12 @@
-#!/bin/ksh -x
+#!/bin/sh -x
 
 ###############################################################
 # Source FV3GFS workflow modules
 
-. $HOMEgfs/ush/load_fv3gfs_modules.sh
+##. $HOMEgfs/ush/load_fv3gfs_modules.sh
+. $HOMEgfs/versions/run.ver
+
+. $HOMEgfs/modulefiles/module_base.wcoss2
```

Comment:
Change this back to the load_fv3gfs_module.sh line. I have updates coming in with my PR into feature/ops-wcoss2 that update and correct the module_base loading at runtime from load_fv3gfs_module.sh. I'm hoping to submit that PR first and then you can update your fork branch with those updates.

Reply:
Went back to loading the original  load_fv3gfs_module.sh. I'll wait for your commit and then pull it  to my branch.